app/views/environments/welcome.html.erb

```diff
@@ -11,4 +11,7 @@
   <div class="blank-slate-pf-main-action">
       <%= new_link(_('New Puppet environment'), {}, { :class => "btn-lg" }) %>
   </div>
+  <div class="blank-slate-pf-secondary-action">
+    <%= select_action_button( _('Import'), {}, import_proxy_links(hash_for_import_environments_environments_path, 'btn btn-default')) %>
```

Comment:
We already have a `import_proxy_select` helper that does this (and is used in the index page for the same button)